### PR TITLE
feat: implement API call for getting a party code - party name dictionary

### LINKED ISSUE #24: feat: add API call for a dictionary from party code to party name
The API should have a way to return a dictionary of party code - party names for all parties in all elections.

### PR DESCRIPTION
**Pull request by Log234**
Implements an API call to return a dictionary which uniquely maps party codes to party names for all parties in all elections.

closes #24

### CODE REVIEW

File: Lavinia-api/Models/Party.cs [hunk 1]
```diff
@@ -6,12 +6,7 @@ namespace LaviniaApi.Models
...
     public class Party
     {
-        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
-        public int PartyId { get; set; }
-
         [Required] public string Name { get; set; }
-
-        public string InternationalName { get; set; }
-        public string ShortName { get; set; }
+        [Required] public string Code { get; set; }
```
Comment (simendag) [inline Lavinia-api/Models/Party.cs:10]: Won't this break API v1?

Comment (Log234) [reply Lavinia-api/Models/Party.cs:10]: No, this model was apparently never used in API v1.

Comment (simendag) [reply Lavinia-api/Models/Party.cs:10]: Then we might want to remove the comment about API v1.

Comment (Log234) [reply Lavinia-api/Models/Party.cs:10]: Updated the comment in a new commit.

**Comment by simendag** [comment]
Will Codacy complete here, or should we force a merge?

**Comment by Log234** [comment]
I will close this PR as this has become a duplicate of #23.

**Comment by Log234** [comment]
Duplicate of #23

**Pull request closed without merging**